src/main/java/com/uber/cadence/internal/shadowing/ReplayWorkflowActivityImpl.java

```diff
@@ -114,21 +111,22 @@ public ReplayWorkflowActivityResult replay(ReplayWorkflowActivityParams request)
       successCount += oneReplayResult.getSucceeded();
       failedCount += oneReplayResult.getFailed();
       skippedCount += oneReplayResult.getSkipped();
-      ReplayWorkflowActivityResult heartbeatResult =
-          new ReplayWorkflowActivityResult()
-              .setSucceeded(successCount)
-              .setFailed(failedCount)
-              .setSkipped(skippedCount);
+      ReplayWorkflowActivityResult heartbeatResult = new ReplayWorkflowActivityResult();
```

Comment:
nit: looks like we can reuse `heartbeatResult` defined above? no strong opinion on this, feel free to ignore.

Reply:
updated